src/bio.c

```diff
@@ -2994,7 +2994,9 @@ int wolfSSL_BIO_flush(WOLFSSL_BIO* bio)
             }
 
         #ifndef NO_FILESYSTEM
-            if (bio->type == WOLFSSL_BIO_FILE && bio->shutdown == BIO_CLOSE) {
+            if (bio->type == WOLFSSL_BIO_FILE &&
+                bio->shutdown == WOLFSSL_BIO_CLOSE)
+            {
```

Comment:
Move brace up to previous line.

Reply:
the typography is much clearer the way I have it -- I use this technique routinely to avoid visual running together of the condition and the enclosed statements.